### PR TITLE
Atom size and mass

### PR DESCRIPTION
**Pull request by TemporalOroboros**
## About The Pull Request
Generalizes the item `w_class` and mob `mob_size` systems to atoms in general.
Honestly, I just wanted to get rid of the `held_mob_size` variable and I thought it was silly we had two different size systems.
Scope creep is pain.

TODO:
- [ ] Figure out how to correctly rescale everything that was scaling off of item `w_class` and mob `mob_size`.
  - Most of them assumed that it represented both mass _and_ volume so that will need to be corrected.
- [ ] Bikeshed the mass and volume of all __13433__ `/atom` subtypes.
- [ ] Bikeshed all of the storage capacities for everything that can store things since item sizes are no longer just 6 evenly spaced values.

## Why It's Good For The Game
Turns two separate systems that did the same thing into one more general system that does the thing.
Makes the code more consistent and easier to expand.

## Changelog

:cl:
code: Items and mobs now use the same size system.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
NOT AGAINNNNNNNNNNN

**Comment by SuperNovaa41** [comment]
ok whens this one getting RED LABEL

**Comment by cacogen** [comment]
how come turfs are 1m x 2.5m when they're square? you could have 1 pixel = 1/32 of a metre if it was 1:1.

**Comment by TemporalOroboros** [comment]
> how come turfs are 1m x 2.5m when they're square? you could have 1 pixel = 1/32 of a metre if it was 1:1.

They aren't. They have three sides and the 2. 5m side is vertical.

File: code/datums/brain_damage/mild.dm [hunk 1]
```diff
@@ -141,7 +141,7 @@
 	else if(owner.get_active_held_item())
 		var/drop_chance = 1
 		var/obj/item/I = owner.get_active_held_item()
-		drop_chance += I.w_class
+		drop_chance += max(round(log(10, I.atom_size) - 2), 0)
```
Comment (Ryll-Ryll) [inline code/datums/brain_damage/mild.dm:144]: This seems infinitely more obnoxious to work with than the defines being 1 2 3 4

**Comment by Ryll-Ryll** [comment]
...what exactly is the benefit of this? None of these numbers or increases in complexity seem very intuitive

**Comment by LemonInTheDark** [comment]
for the record this worries me, since it moves size away from something well defined, and into random number territory. Also makes it need to be more accurate to sprite size, which is concerning

**Comment by Ryll-Ryll** [comment]
I'm gonna go and close this tomorrow if there's not a good reason for pushing this forward

**Pull request closed without merging**